iothub_client/tests/iothubtr_amqp_tel_msgr_ut/iothubtr_amqp_tel_msgr_ut.c

```diff
@@ -1665,11 +1665,109 @@ static TELEMETRY_MESSENGER_HANDLE create_and_start_messenger2(TELEMETRY_MESSENGE
     return handle;
 }
 
+char* umock_stringify_BINARY_DATA()
+{
+    char* result = (char*)TEST_malloc(1);
+    result[0] = '\0';
+    return result;
+}
+
+int umock_are_equal_BINARY_DATA()
+{
+    //force fall through to success bypassing access violation
+    int result = 1;
+    return result;
+}
+
+int umock_copy_BINARY_DATA()
+{
+    //force fall through to success bypassing access violation
+    int result = 0;
+    return result;
+}
+
+void umock_free_BINARY_DATA()
+{
+    //do nothing
+}
+
+char* umock_stringify_TELEMETRY_MESSENGER_MESSAGE_DISPOSITION_INFO(const TELEMETRY_MESSENGER_MESSAGE_DISPOSITION_INFO* value)
+{
```

Comment:
Same comments here are above for AMQP disposition.

Reply:
fixed : mirrored the AMQP Disposition